package/com.unity.formats.usd/Dependencies/USD.NET.Unity/Geometry/MeshSampleBase.cs

```diff
@@ -37,21 +36,12 @@ public class MeshSampleBase : PointBasedSample
         /// When not explicitly specified by the shader, "st" should be considered the default uv set.
         /// </summary>
         /// <remarks>
-        /// UV object types should be Vector{2,3,4}[], List of Vector{2,3,4}, or null.
+        /// UV object types should be Vector{2,3}[], List of Vector{2,3}, or null.
```

Comment:
Why just 2,3 components?

Reply:
Because I could not find any reference to TexCoord4f. What are UV coords of type float4 by the way?

Reply:
Unity has float4 uvs, which are useful sometimes for storing tangent data, etc